tjhsst/static/style.css

```diff
@@ -46,7 +46,7 @@ html, body {
     height: 24px;
     background-color: rgb(0, 80, 105);
     border-radius: 3px;
-    margin-left: 1rem;
+    margin: .5rem 0rem 0rem 1rem;
```

Comment:
The main navbar's search field is also selected by `.navbar .search`, so this disturbs the vertical alignment when *not* on mobile:
![image](https://user-images.githubusercontent.com/23221704/59986526-3a61c680-9605-11e9-9f15-919399a9933d.png)
Please create a separate declaration (perhaps with a selector of something like `.navbar .navbar-collapse .search`) to set the `margin-top`.